docs/en/jpql-with-kotlin-jdsl/custom-dsl.md

```diff
@@ -39,6 +42,35 @@ val query = jpql(MyJpql) {
 }
 ```
 
+### Jpql Instance
+
+With this method, you can reuse a single instance for query creation and utilize dependency injection.
```

Comment:
The use of `way` in the introduction and `method` in the details doesn't seem to fit together. 

How about changing `method` to `way`?

```suggestion
With this way, you can reuse a single instance for query creation and utilize dependency injection.
```